src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```diff
@@ -33,17 +33,17 @@ api_platform:
 framework:
     messenger:
         buses:
-            sylius_default.bus:
-                middleware:
-                    - 'validation'
-                    - 'doctrine_transaction'
-            sylius_event.bus:
-                middleware:
-                    - 'validation'
-                    - 'doctrine_transaction'
```

Comment:
Are these removed middlewares configured somewhere?

Reply:
Only event bus middlewares were meant to be removed. Event bus should be used only to inform the rest of the system about already happened stuff, so there is no need for transaction nor validation. Any new logic should be put on the command bus again if some part of the system would like to react to the event